_posts/2016-10-17-test-stream.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: post
+title:  "Testing Unbounded Pipeliens in Apache Beam"
+date:   2016-10-17 10:21:48 -0800
+excerpt_separator: <!--more-->
+categories: blog beam testing java
```

Comment:
Don't add all these. They make the URL weird. I'd just stick to 'blog'. Also filed BEAM-776 to figure out how to make this not ugly.